src/i18n/de/README.md

```diff
@@ -81,7 +81,7 @@ Diese Anleitung soll dazu beitragen, dass sich das Ergebnis beim Lesen trotz all
 - Falsch geschriebene Markennamen (wir halten uns an die offizielle Schreibweise auf der Hersteller-Website)
 	- Github --> GitHub
 	- Javascript --> JavaScript
-	- Typescript --> TypeScript
+	- TypeScript --> TypeScript
```

Comment:
It is README and don't need fix.
```suggestion
	- Typescript --> TypeScript
```

Reply:
Thanks for spotting this, @liruifengv! Yes, this one should stay "wrong" because it is an example showing that one must correct the "wrong" way of spelling into the correct way!